source/api/extension/navplace/index.md

```diff
@@ -166,17 +166,15 @@ Clients may use this property for navigational purposes on open web map systems.
 
 ### 3.1 Linked Data Context
 
-The GeoJSON-LD context and navPlace extension context  _MUST_ be included before the IIIF Presentation API 3 context on the top-level object.
-
-
 
 *   The URI of the GeoJSON-LD linked data context is [`http://geojson.org/geojson-ld/geojson-context.jsonld`](http://geojson.org/geojson-ld/geojson-context.jsonld)
 *   The URI of the navPlace linked data context extension is [`http://iiif.io/api/extension/navPlace-context/context.json`](http://iiif.io/api/extension/navPlace-context/context.json)
-*   The URI of the IIIF Presentation API 3 linked data context extension is
- [`http://iiif.io/api/extension/navPlace-context/context.json`](http://iiif.io/api/extension/navPlace-context/context.json)
+*   The URI of the IIIF Presentation API 3 linked data context
+ [`https://iiif.io/api/presentation/3/context.json`](https://iiif.io/api/presentation/3/context.json)
 
+The navPlace extension context  _MUST_ be included before the IIIF Presentation API 3 context on the top-level object.  It is worth noting that the navPlace extension context file includes the GeoJSON-LD context through context scoping and so the GeoJSON-LD context does not have to be explicitly included on the top level object, unless GeoJSON is used in other properties besides navPlace.  
```

Comment:
Should navPlace at the end of the sentence be `navPlace` since it is referencing the property?

Reply:
Very likely, yes.  Need to have a formatting pass for these kinds of things.